src/components/LineChart/index.tsx

```diff
@@ -0,0 +1,177 @@
+import { useTheme } from '@emotion/react'
+import { DatumValue, ValueFormat } from '@nivo/core'
+import { LineSvgProps, Point, ResponsiveLine, Serie } from '@nivo/line'
+import PropTypes from 'prop-types'
+import React, { FunctionComponent, Validator, useState } from 'react'
+import CustomLegend, { Transformer } from './CustomLegend'
+import LineChartTooltip from './Tooltip'
+import { getLegendColor, getMaxChartValue, getMinChartValue } from './helpers'
+
+type Formater = ValueFormat<DatumValue>
+
+type Shape<K extends string, T> = {
+  [P in K]?: T
+}
+
+type LineChartProps = {
+  height?: string | number
+  margin?: Shape<'bottom' | 'left' | 'right' | 'top', number>
```

Comment:
bottom, left, right, top possible to export it into a Position type ?

Reply:
Changed to Box (imported from `@nivo/core`)